conf/application.conf

```diff
@@ -122,6 +123,7 @@ features{
   scp08-enabled = true
   release-c-enabled = false
   run-remove-email-addresses-job-enabled = false
+  drop-undertaking-cache-job-enabled = false
```

Comment:
I think it's safe to directly set it to `true` here, rather than in the app-config repos. What do you think @habeeb01?